airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryDestination.java

```diff
@@ -149,11 +149,7 @@ public AirbyteConnectionStatus checkGcsPermission(final JsonNode config) {
       message.append(" Please make sure the service account can access the bucket path, and the HMAC keys are correct.");
 
       LOGGER.error(message.toString(), e);
-
-      return new AirbyteConnectionStatus()
-          .withStatus(AirbyteConnectionStatus.Status.FAILED)
-          .withMessage("Could access the GCS bucket with the provided configuration.\n" + e
-              .getMessage());
+      throw new ConfigErrorException("Could access the GCS bucket with the provided configuration.\n", e);
```

Comment:
It seems this message should be "Could **not** access the GCS bucket with the provided configuration.\n", right because we're saying that it failed

Reply:
Hm... Interesting fact, it's been there for already almost half a year. Updated, thanks